Index.html

```diff
@@ -1,269 +1,176 @@
+  <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+  <script src="https://kit.fontawesome.com/bee9655978.js" crossorigin="anonymous"></script>
+  <meta name="Portfolio" content="this is a portfolio by Mohammad Mehdi Pazouki.">
+
+  <title>Mohammad Mehdi Pazouki</title>
 </head>
-<body>
-    <header class="position__nav toolbar">
-        <div class="corner__geometry  mobile__header">
-            <img class="mobile__visible" src="./Contents/Resources/Pictures/image_geometry_1.svg" alt="">
-            <img class="mobile__invisible" src="./Contents/Resources/Pictures/image_geometry_1.svg" alt="">
-        </div>
-        <!-- only add the toolbar__open__rowx using javascript -->
-        <nav class="drop-down menu__button__align mobile__header">
-            <button id="hamburger">
-                <i class="fa fa-bars closed__menu"></i>
-                <img class="opened__menu cross" src="./Contents/Resources/Icons/cross.svg" alt="">
-            </button>
-        </nav>
-
-        <img class="mobile__photos mobile__invisible" src="./Contents/Resources/Icons/nav_dots.svg" alt="">
 
-        <ul class="mobile__invisible desktop__header ">
-            <li class="header__navication"><a class="nav__desktoplink" href="#headline">Hello</a></li>
-            <li class="header__navication"><a class="nav__desktoplink" href="#works">Works</a></li>
-            <li class="header__navication"><a class="nav__desktoplink" href="#about">About</a></li>
-            <li class="header__navication"><a class="nav__desktoplink" href="#contact">Contact</a></li>
+<body>
+  <header id="header">
+    <nav id="header_nav">
+      <i class="fa-solid fa-bars" id="hamburger-menu"></i>
+      <ul id="desktop-nav">
+        <li>
+          <a href="#intro">Hello</a>
+        </li>
+        <li>
+          <a href="#Projects">Portfolio</a>
+        </li>
+        <li>
+          <a href="#about">About</a>
+        </li>
+        <li>
+          <a href="#contact-form">contact</a>
+        </li>
+      </ul>
+      <i class="fa-solid fa-xmark" id="closeMenuBtn"></i>
+      <ul id="mobile-nav">
+        <li>
+          <a href="#intro">Hello</a>
+        </li>
+        <li>
+          <a href="#Projects">Portfolio</a>
+        </li>
+        <li>
+          <a href="#about">About</a>
+        </li>
+        <li>
+          <a href="#contact-form">contact</a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+  <section id="intro">
+    <div>
+      <h1>Hi, I'm <b>Mohammad Mehdi Pazouki</b>, Happy to see you</h1>
+      <p>
+        Hello! I am a software developer! I can help you build a product,
+        feature or website. Take a look of my works. If you like what you see
+        and have a project you need coded, don't hesitate and contact me.
+      </p>
+      <button type="button" class="dark small">Start collaboration</button>
+      <div id="logo"></div>
+    </div>
+  </section>
```

Comment:
- [ ] Kindly consider giving your elements exact margins, paddings, font-sizes and font-weights as it can be seen in the figma design because we need to follow it as per requirements. kindly have a look at the positioning of your elements and positioning of elements in figma design. kindly consider fixing this elements positioning issue.
  - Also for example the font size of the p tag is bigger in your design so consider fixing it.
your deisgn 
![r](https://user-images.githubusercontent.com/96683843/236262870-8bde7d2d-4af2-4b52-a5a8-872650d6d5b6.jpg)

figma design
![f](https://user-images.githubusercontent.com/96683843/236262929-1b725f64-77dc-4395-805c-3e25c997f5dd.jpg)